nessie/src/test/java/org/apache/iceberg/nessie/TestNamespace.java

```diff
@@ -73,6 +77,53 @@ public void testListNamespaces() {
     Assertions.assertThat(namespaces).isNotNull().hasSize(2);
   }
 
+  @Test
+  @NessieApiVersions(versions = NessieApiVersion.V1)
+  public void testNamespacesWithV1Api() {
+    createNamespacesAndTables();
+
+    testListTables();
+
+    List<Namespace> namespaces;
+    namespaces = catalog.listNamespaces(Namespace.of("foo"));
+    // NessieApiV1 should include both implicit and explicit namespaces
+    Assertions.assertThat(namespaces)
+        .isNotNull()
+        .hasSize(2)
+        .isEqualTo(Arrays.asList(Namespace.of("foo"), Namespace.of("foo", "bar")));
```

Comment:
Is the list order guaranteed? Why not `.containsExactlyInAnyOrder`?

Reply:
done.